src/components/Card/src/Card/Card.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { createRef } from 'react';
+import MaterialCard from '@material-ui/core/Card';
+import { StylesProvider } from '@material-ui/core/styles';
+import { ArrowRightIcon } from '@gemeente-denhaag/icons';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import './mui-override.css';
+import './card.css';
+import { Typography } from '@gemeente-denhaag/typography';
+import { cardClasses, cardArrowClasses, cardTitleClasses, cardSubtitleClasses, cardCaseClasses } from './bem-mapping';
+import { CardContent } from '../CardContent/CardContent';
+import { CardActions } from '../CardActions/CardActions';
+import { Icon } from '@material-ui/core';
+
+export interface CardProps extends BaseProps {
+  /**
+   * Callback fired when the Card is clicked.
+   */
+  onClick?: (event: React.MouseEvent<HTMLDivElement>) => void;
+
+  /**
+   * Determines the variant of the card
+   * Variant can be either `basic` or `case`
+   */
+  variant?: 'basic' | 'case';
+
+  /**
+   * Determines the title of the card
+   */
+  title: string;
+
+  /**
+   * Determines the subtitle of the card
+   */
+  subTitle?: string;
+
+  /**
+   * Determines the date shown on the card
+   */
+  date: Date;
+
+  /**
+   * Determines the url the card points to
+   */
+  href: string;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Card: React.FC<CardProps> = (props: CardProps) => {
+  let classes;
+  const arrowClasses = cardArrowClasses;
+  const titleClasses = cardTitleClasses;
+  const subtitleClasses = cardSubtitleClasses;
+  const linkRef = createRef<HTMLAnchorElement>();
+
+  const title = (
+    <a href={props.href} ref={linkRef}>
+      {props.title}
+    </a>
+  );
+
+  const onClick = () => {
+    if (linkRef.current !== null) {
+      linkRef.current.click();
+    }
+  };
+
+  switch (props.variant) {
+    case 'case':
+      classes = cardCaseClasses;
+      break;
+    case 'basic':
+    default:
+      classes = cardClasses;
+  }
+
+  return (
+    <StylesProvider injectFirst>
+      <MaterialCard classes={classes} onClick={onClick}>
+        <div className="denhaag-card__wrapper">
+          <div className="denhaag-card__background"></div>
+          <CardContent>
+            <div className="denhaag-card__text-wrapper">
+              <Typography classes={titleClasses} component="p">
+                {title}
+              </Typography>
```

Comment:
Nice use of the `<a>` tag, looks like this is a solid setup wrt a11y!
Two questions:
- What made you decide to place the HTML of the `<a>`-tag in a variable instead of just right here?
- Might be a bit of a discussion point whether or not a `<p>` HTML tag is actually semantically correct here. As the name implies `<p>` is a `paragraph` tag, used to semantically markup a piece of text as a paragraph. In my eyes a single line of text for a title or a subtitle does not make a paragraph which makes me think this is not actually semantically correct HTML use. Not sure what the best approach would be in this construction though as the `Typography` component is necessary for the styling. My first thought is to use a extra `<div>` for the block, with a `Typography` component which uses a HTML `span`.

Reply:
For the first point, I think this helps to make clear where the react Ref is used and we should stick to this.
For the second point, in the future we should refactor CardCase to make use of the  updated Typography component from our library. We can move the discussion to that refactoring.
https://github.com/Gemeente-DenHaag/denhaag-component-library/issues/253